lxd/instance/drivers/driver_qemu.go

```diff
@@ -4964,3 +4964,47 @@ func (d *qemu) writeInstanceData() error {
 
 	return nil
 }
+
+// Info returns "qemu" and the currently loaded qemu version.
+func (d *qemu) Info() instance.Info {
+	var qemuVersion, qemuBinary, qemuPath string
+	var out []byte
+	var err error
+	data := instance.Info{
+		Name:    "",
+		Version: "",
+	}
+
+	if d.architecture == 0 {
+		d.architecture, err = osarch.ArchitectureGetLocalID()
+	}
+	if err != nil {
+		return data
+	}
+
+	qemuBinary, _, err = d.qemuArchConfig()
+	if err != nil {
+		return data
+	}
+
+	qemuPath, err = exec.LookPath(qemuBinary)
+	if err != nil {
+		return data
+	}
+
+	out, err = exec.Command(qemuPath, "--version").Output()
+	if err != nil {
+		return data
+	}
+
+	data.Name = "qemu"
```

Comment:
Can be dropped per above comment to just immediately declare `Name` as `qemu` and having logic in `createInstanceDriversCache` to skip the Info structs with an empty version string.

Reply:
I was about to ask where you'd want me to handle the empty driver versions, but that works

Reply:
Fixed in 37c1ac5